building/upload-debs/rebuild.sh

```diff
@@ -1,7 +1,25 @@
 #!/bin/bash
 set -e -u
 
+if [ -z "${APT_REPOSITORY:-}" ]
+then
+	echo "Need to specify apt repository:"
+	echo "\$ export APT_REPOSITORY=<username>/<branch>"
+	exit 1
+fi
+
+if [ -z "${APT_SIGNING_KEY:-}" ]
+then
+	echo "Need to specify apt signing key:"
+	echo "\$ export APT_SIGNING_KEY=<key identifier>"
+	exit 1
+fi
+
+sed "s;\\\${APT_SIGNING_KEY};${APT_SIGNING_KEY};g" < conf/distributions.in > conf/distributions
+
 cd "$(dirname "$0")"
-reprepro -Vb . includedeb homeworld ../build-debs/binaries/homeworld-*.deb
-rsync -av --progress --delete-delay ./dists ./pool /mit/hyades/debian/
+gpg --import default-repo-signing-key.gpg
+reprepro -Vb . update
```

Comment:
Does `update` correctly handle the situation where we're trying to publish to the base repository?

Reply:
In theory, it should -- it only pulls updates if the version number is less than your currently built version, so you would have the updated debs in your repository when you rsync them over. It would be better to test this though -- could you do it just before merge? I don't have the bits for that

Reply:
Yeah, I can test this once we're closer to merge.

Have we confirmed that `update` verifies signatures properly?